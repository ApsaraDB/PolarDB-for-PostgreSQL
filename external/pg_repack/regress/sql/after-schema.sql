--
-- tables schema after running repack
--

\d tbl_cluster
\d tbl_gistkey
\d tbl_only_ckey
\d tbl_only_pkey
\d tbl_incl_pkey
\d tbl_with_dropped_column
\d tbl_with_dropped_toast
\d tbl_idxopts
