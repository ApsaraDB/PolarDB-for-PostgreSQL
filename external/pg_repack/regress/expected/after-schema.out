--
-- tables schema after running repack
--
\d tbl_cluster
                      Table "public.tbl_cluster"
 Column |            Type             | Collation | Nullable | Default 
--------+-----------------------------+-----------+----------+---------
 col1   | integer                     |           | not null | 
 time   | timestamp without time zone |           |          | 
 ,")    | text                        |           | not null | 
Indexes:
    "tbl_cluster_pkey" PRIMARY KEY, btree (","")", col1) WITH (fillfactor='75')
    ",") cluster" btree ("time", length(","")"), ","")" text_pattern_ops) WITH (fillfactor='75') CLUSTER

\d tbl_gistkey
            Table "public.tbl_gistkey"
 Column |  Type   | Collation | Nullable | Default 
--------+---------+-----------+----------+---------
 id     | integer |           | not null | 
 c      | circle  |           |          | 
Indexes:
    "tbl_gistkey_pkey" PRIMARY KEY, btree (id)
    "cidx_circle" gist (c) CLUSTER

\d tbl_only_ckey
                     Table "public.tbl_only_ckey"
 Column |            Type             | Collation | Nullable | Default 
--------+-----------------------------+-----------+----------+---------
 col1   | integer                     |           |          | 
 col2   | timestamp without time zone |           |          | 
 ,")    | text                        |           |          | 
Indexes:
    "cidx_only_ckey" btree (col2, ","")") CLUSTER

\d tbl_only_pkey
           Table "public.tbl_only_pkey"
 Column |  Type   | Collation | Nullable | Default 
--------+---------+-----------+----------+---------
 col1   | integer |           | not null | 
 ,")    | text    |           |          | 
Indexes:
    "tbl_only_pkey_pkey" PRIMARY KEY, btree (col1)

\d tbl_incl_pkey
                     Table "public.tbl_incl_pkey"
 Column |            Type             | Collation | Nullable | Default 
--------+-----------------------------+-----------+----------+---------
 col1   | integer                     |           | not null | 
 col2   | timestamp without time zone |           |          | 
Indexes:
    "tbl_incl_pkey_pkey" PRIMARY KEY, btree (col1) INCLUDE (col2)

\d tbl_with_dropped_column
      Table "public.tbl_with_dropped_column"
 Column |  Type   | Collation | Nullable | Default 
--------+---------+-----------+----------+---------
 c1     | text    |           |          | 
 id     | integer |           | not null | 
 c2     | text    |           |          | 
 c3     | text    |           |          | 
Indexes:
    "tbl_with_dropped_column_pkey" PRIMARY KEY, btree (id) WITH (fillfactor='75') CLUSTER
    "idx_c1c2" btree (c1, c2) WITH (fillfactor='75')
    "idx_c2c1" btree (c2, c1)

\d tbl_with_dropped_toast
       Table "public.tbl_with_dropped_toast"
 Column |  Type   | Collation | Nullable | Default 
--------+---------+-----------+----------+---------
 i      | integer |           | not null | 
 j      | integer |           | not null | 
Indexes:
    "tbl_with_dropped_toast_pkey" PRIMARY KEY, btree (i, j) CLUSTER

\d tbl_idxopts
            Table "public.tbl_idxopts"
 Column |  Type   | Collation | Nullable | Default 
--------+---------+-----------+----------+---------
 i      | integer |           | not null | 
 t      | text    |           |          | 
Indexes:
    "tbl_idxopts_pkey" PRIMARY KEY, btree (i)
    "idxopts_t" btree (t DESC NULLS LAST) WHERE t <> 'aaa'::text

